consumer-configuration/README.adoc

```diff
@@ -0,0 +1,347 @@
+----
+
+The `<bootstrap_server>` is the bootstrap server endpoint for your service account.
+You copy this information for the Kafka instance in the {product} web console by selecting the options menu (three vertical dots) and clicking *View connection information*.
+
+ifndef::community[]
+NOTE: The Kafka bin scripts are part of the open source community version of Apache Kafka. The bin scripts are not a part of {product} and are therefore not supported by Red Hat.
+endif::[]
+
+.Prerequisites
+* You are logged in to the {product} web console.
+* You have created a Kafka instance with at least one Kafka topic in {product}.
+* Consumer client applications connected to the Kafka instance have a consumer group ID.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the consumer groups that you want to review.
++
+Alternatively, select a consumer group for a specific topic.
++
+--
+.. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the topic.
+.. In the *Topics* page, click the name of the topic.
+--
+
+. In the *Consumer groups* page, click the name of the consumer group that you want to review.
++
+Alternatively, select the options icon (three vertical dots) for the relevant consumer group and click *View partition offsets*.
+
+[role="_additional-resources"]
+.Additional resources
+* {base-url}{rhoas-cli-url}[_Getting started with the `rhoas` CLI_^]
+* {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^]
+
+[id="ref-supported-consumer-group-properties_{context}"]
+== Consumer group properties presented in {product}
+
+[role="_abstract"]
+The following consumer group properties are presented in {product}.
+Consumer group properties are used for monitoring in the {product} web console and are not editable.
+
+=== Consumer groups
+
+Consumer Group ID::
+The consumer group ID is the unique identifier for the consumer group within the cluster. This is part of the consumer configuration for the application client.
+
+Active Members::
+Active members shows the number of consumers in the group that are assigned to a topic partition in the Kafka instance.
+If you are viewing information on consumer groups for a topic, these are the active members for the topic.
+
+Partitions with lag::
+Partitions with lag shows the number of partitions where the assigned consumer has not caught up with the last message in the partition.
+The lag reflects the position of the consumer offset in relation to the end of the partition log.
+
+=== Consumer offset positions
+
+When you click on a consumer group, you see the details of each member.
+
+Partition::
+The partition number for the topic.
+
+Client ID::
+The unique ID of the client application used to identify active consumers.
+If there is no client ID shown, the partition is not currently being consumed.
```

Comment:
"If no client ID is shown, ..."  This to avoid the empty "there is/are", which IBM SG encourages to avoid. This comes up a few times, so worth searching all instances of "there is" and "there are" to address. Also applies to empty "it is", like "it is important to ...". Don't know if that applies to this doc, but just noting.

Reply:
I think this is the only one. Updated.